src/test/resources/pacts/publicapi-connector-search-refunds-with-from-and-to-date.json

```diff
@@ -148,28 +148,28 @@
                 }
               ]
             },
-            "$.results[0].links[0].href": {
+            "$.results[0]._links[0].href": {
```

Comment:
did you really mean to change these? is the response structure from connector changing?

Reply:
Ooops! Fixed, good catch 👍